terraso_backend/apps/graphql/schema/__init__.py

```diff
@@ -95,6 +95,7 @@ class Query(graphene.ObjectType):
     taxonomy_terms = DjangoFilterConnectionField(TaxonomyTermNode)
     story_map = TerrasoRelayNode.Field(StoryMapNode)
     story_maps = DjangoFilterConnectionField(StoryMapNode)
+    sites = DjangoFilterConnectionField(SiteNode, filterset_class=SiteFilter)
```

Comment:
suggestion: can the `SiteFilter` be added via `SiteNode`'s meta class for consistency?

Reply:
@JeancarloBarrios could you address this before merging?